.tekton/release.yaml

```diff
@@ -0,0 +1,94 @@
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  name: cachi2-on-release
+  annotations:
+    pipelinesascode.tekton.dev/max-keep-runs: "3"
+    pipelinesascode.tekton.dev/on-event: "[push]"
+    pipelinesascode.tekton.dev/on-target-branch: "[refs/tags/*]"
+    pipelinesascode.tekton.dev/task: "[git-clone]"
+spec:
+  params:
+    - name: repo_url
+      value: "{{repo_url}}"
+    - name: revision
+      value: "{{revision}}"
+  pipelineSpec:
+    tasks:
+      - name: fetch-repository
+        taskRef:
+          name: git-clone
+        workspaces:
+          - name: output
+            workspace: workspace
+        params:
+          - name: depth
+            value: "0"
+          - name: url
+            value: $(params.repo_url)
+          - name: revision
+            value: $(params.revision)
+      - name: release
+        runAfter:
+          - fetch-repository
+        workspaces:
+          - name: output
+            workspace: workspace
+        taskSpec:
+          results:
+            - name: version
+          workspaces:
+            - name: output
+          steps:
+            - name: get-semver-tag
+              image: registry.access.redhat.com/ubi9/python-39
+              workingDir: $(workspaces.output.path)
+              env:
+              - name: WORKSPACE_OUTPUT_PATH
+                value: $(workspaces.output.path)
+              - name: PARAM_REVISION
+                value: $(params.revision)
+              script: |
+                #!/usr/bin/env bash
+                set -eufx
+                
+                git config --global --add safe.directory "${WORKSPACE_OUTPUT_PATH}"
+                git fetch --tag -v
+                version=$(git  --no-pager tag --points-at HEAD)
+                [[ -z ${version} ]] && {
+                    echo "No tag points at commit $PARAM_REVISION"
+                    exit 1
+                }
+
+                if [[ $version =~ ^(0|1)\.(0|[1-9][0-9]*)\.(0|[1-9][0-9]*)$  ]]; then
```

Comment:
Does this mean we're not confident we'll have a Cachi2 2 release? :smiling_face_with_tear: 
(anything >=2.0.0, I mean)

Reply:
I'm not :D What would you like to include there?

Reply:
We could change the first part of the regex to accept major versions besides 0 and 1. Probably not a scenario we'll see anytime soon, but the future generations might be puzzled on why their 2.0.0 tag is failing.

Reply:
@brunoapimentel it will work up to version 9.x.x now :)